os-partition/added/partitionPV.sh

```diff
@@ -48,6 +48,8 @@ function partitionPV() {
           SERVER_DATA_DIR="${INSTANCE_DIR}/serverData"
           mkdir -p "${SERVER_DATA_DIR}"
 
+          init_data_dir ${SERVER_DATA_DIR}
```

Comment:
The check for initialised data is a good idea, would it make more sense to move the check here?  It would not then be needed in the init_data_dir function.

if [ ! -f "${SERVER_DATA_DIR}/../data_initialized" ]; then
      init_data_dir "${SERVER_DATA_DIR}"
      touch "${SERVER_DATA_DIR}/../data_initialized"
fi